openshift/mlflow-experiment-job-template.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: Template
+
+labels:
+  application: ${APPLICATION_NAME}
+
+metadata:
+  name: mlflow-experiment-job
+
+parameters:
+- description: Application name
+  from: "experiment-run-[a-z0-9]{4}"
+  generate: expression
+  name: APPLICATION_NAME
+  required: true
+- name: MLFLOW_TRACKING_URI
+  value: http://$(MLFLOW_SERVER_SERVICE_SERVICE_HOST):$(MLFLOW_SERVER_SERVICE_SERVICE_PORT) # To use the internal host and port number to connect to the mlflow tracking server
+  required: true
+- name: APP_IMAGE_URI
+  description: "Name of the application image to be pulled from the registry"
+  value: "mlflow-experiment-image"
+  required: true
+- name: APP_IMAGE_TAG
+  value: "latest"
+  required: true
+
+
+objects:
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: ${APPLICATION_NAME}
+    labels:
+      app: ${APPLICATION_NAME}
+      experiment: ${APP_IMAGE_URI}
+  spec:
+    template:
+      metadata:
+        name: ${APPLICATION_NAME}
+      spec:
+        containers:
+        - name: ${APPLICATION_NAME}
+          image: ${APP_IMAGE_URI}:${APP_IMAGE_TAG}
+          imagePullPolicy: Always
+          resources:
+            requests:
+              memory: 500Mi
+            limits:
```

Comment:
can you make the `limits` part configurable? 
And add a configurable limit for `cpu`, with a default of `1`